cli/npm/managed/registry.rs

```diff
@@ -217,30 +221,34 @@ impl CliNpmRegistryApiInner {
     }
   }
 
-  fn load_file_cached_package_info_result(
+  async fn load_file_cached_package_info_result(
     &self,
     name: &str,
   ) -> Result<Option<NpmPackageInfo>, AnyError> {
     let file_cache_path = self.get_package_file_cache_path(name);
-    let file_text = match fs::read_to_string(file_cache_path) {
-      Ok(file_text) => file_text,
-      Err(err) if err.kind() == ErrorKind::NotFound => return Ok(None),
-      Err(err) => return Err(err.into()),
-    };
-    match serde_json::from_str(&file_text) {
-      Ok(package_info) => Ok(Some(package_info)),
-      Err(err) => {
-        // This scenario might mean we need to load more data from the
-        // npm registry than before. So, just debug log while in debug
-        // rather than panic.
-        log::debug!(
-          "error deserializing registry.json for '{}'. Reloading. {:?}",
-          name,
-          err
-        );
-        Ok(None)
+    let name = name.to_string();
```

Comment:
Minor thing, but the clone here can be avoided: https://github.com/denoland/deno/pull/24019/files#diff-78e33d968f4c32d73a023d0c5ce355de065cb8baec42916067f38fac100cb435R222-R235